src/main/java/uk/gov/justice/probation/courtcasematcher/controller/SqsAdminController.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.justice.probation.courtcasematcher.controller;
+
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.justice.probation.courtcasematcher.service.SqsAdminService;
+
+@RestController
+public class SqsAdminController {
+
+    private SqsAdminService sqsAdminService;
+
+    public SqsAdminController(SqsAdminService sqsAdminService) {
+        this.sqsAdminService = sqsAdminService;
+    }
+
+    @PutMapping("/retry-all-dlqs")
+    public void replayDlqMessages() {
```

Comment:
Will need auth on this if it isn't configured elsewhere. Example [here](https://github.com/ministryofjustice/hmpps-spring-boot-sqs/blob/2fa3ca29f8784d99b5e1d798a47557385c05def8/hmpps-sqs-spring-boot-autoconfigure/src/main/kotlin/uk/gov/justice/hmpps/sqs/HmppsQueueResource.kt#L16)

Reply:
Raised [PIC-2052](https://dsdmoj.atlassian.net/browse/PIC-2052) to follow up with this